test/conformance/chainsaw/validate/clusterpolicy/standard/apicalls/default/chainsaw-step-01-apply-1-1.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kyverno.io/v2beta1
+kind: ClusterPolicy
+metadata:
+  name: check-subjectaccessreview
+spec:
+  background: false
+  rules:
+  - context:
+    - apiCall:
+        data:
+        - key: kind
+          value: SubjectAccessReview
+        - key: apiVersion
+          value: authorization.k8s.io/v1
+        - key: spec
+          value:
+            resourceAttributes:
+              group: ""
+              name: '{{ request.namespace }}'
+              resource: namespaces
+              verb: delete
+            user: system:serviceaccount:kyverno:kyverno-admission-controller
+        method: POST
+        urlPath: /apis/authorization.k8s.io/v1/subjectaccessreview
+        default: ''
```

Comment:
Can you add a unit test where the default value is being used, something where the url is invalid

Reply:
The URL which I have added in chainsaw test is invalid. Instead of `apis/authorization.k8s.io/v1/subjectaccessreviews`, I have used `apis/authorization.k8s.io/v1/subjectaccessreview` which is an invalid one.

Reply:
Okay, the default value is empty string and you are matching that with true, imo its not clear if that test works.

Can you use a URL that is clearly invalid and then verify that the default value is returned in the conditions